metrics.go

```diff
@@ -423,77 +428,77 @@ func registerMetrics() {
 }
 
 // report reports metrics to prometheus based on rocksdb props and stats
-func (m *Metrics) report(props *properties, stats *stats) {
+func (m *Metrics) report(dbName string, props *properties, stats *stats) {
```

Comment:
Is there a list/documentation of which metrics are reported? Can we link to it in Confluence?

Reply:
Best sources of information:
- grafana dashboard
- source code
- rocksdb wiki

but I'll add a README in a next PR

Reply:
README is added https://github.com/Kava-Labs/opendb/tree/yevhenii/ethermint?tab=readme-ov-file#purpose-of-this-module